.github/workflows/release-init.yml

```
@@ -0,0 +1,19 @@
+  create-release-branch:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+
+      - name: Create release branch
+        uses: peterjgrainger/action-create-branch@v3.0.0
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          branch: releases/v.${{ inputs.release-version }}
```

actionlint (1.7.4)

property "release-version" is not defined in object type {}

(expression)